dev/com.ibm.ws.javaee.ddmodel/test/com/ibm/ws/javaee/ddmodel/app/AppTest.java

```diff
@@ -17,58 +17,55 @@
 import com.ibm.ws.javaee.ddmodel.DDParser;
 import com.ibm.ws.javaee.version.JavaEEVersion;
 
-
 /**
  * Application deployment descriptor parse tests.
  */
 public class AppTest extends AppTestBase {
-    public static final Version[] TEST_VERSIONS =
-        new Version[] { 
-            JavaEEVersion.VERSION_6_0,
-            JavaEEVersion.VERSION_7_0,
-            JavaEEVersion.VERSION_8_0
-        };
+    public static final Version[] TEST_VERSIONS = new Version[] {
+                                                                  JavaEEVersion.VERSION_6_0,
+                                                                  JavaEEVersion.VERSION_7_0,
+                                                                  JavaEEVersion.VERSION_8_0
+    };
```

Comment:
Should this test be updated to cover VERSION_9_0?  It looks like that was missed when that version was added to JavaEEVersion back in July.

Reply:
done